src/openms_gui/source/VISUAL/Plot1DCanvas.cpp

```diff
@@ -2089,7 +2089,7 @@ namespace OpenMS
   {
     if (intensity_mode_ == IM_PERCENTAGE)
     {
-      percentage_factor_ = overall_data_range_.maxPosition()[1] / getLayer(layer_index).getCurrentSpectrum().getMaxInt();
+      percentage_factor_ = overall_data_range_.maxPosition()[1] / getLayer(layer_index).getCurrentSpectrum().getMaxIntensity();
```

Comment:
is here maxPosition()[1] still needed?

Reply:
yes, because `overall_data_range_` is still the old-style range. Replacing that is part of an upcoming PR